frontend/src/scenes/events/EventDetails.js

```diff
@@ -8,6 +8,12 @@ import { createActionFromEvent } from './createActionFromEvent'
 const { TabPane } = Tabs
 
 export function EventDetails({ event }) {
+    let displayableEventProperties = {}
+
+    for (let key of Object.keys(event.properties)) {
+        if (key !== '$snapshot_data') displayableEventProperties[key] = event.properties[key]
```

Comment:
I think tim is working on this, but it might be a good usecase for extracting this to an util.